hr_timesheet_role/migrations/12.0.1.0.1/pre-migrate.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(cr, version):
```

Comment:
@pedrobaeza is this correct? If migration is from 12.0.1.0.1, we're copying old field value to the new one introduced by parent module

Reply:
Folder name must be target version, not source version, so `12.0.2.0.0`

Reply:
Thanks, fixed!